tests/acceptance/features/webUISharingPublic/shareByPublicLink.feature

```diff
@@ -98,16 +118,19 @@ Feature: Share by public link
     Then file "lorem.txt" should be listed on the webUI
     And the content of "lorem.txt" on the remote server should be the same as the local "lorem.txt"
 
+  @skipOnOCIS
```

Comment:
also tagging with issue labels would be more descriptive why the scenario is skipped!

Reply:
@kiranparajuli589 there's another issue created https://github.com/owncloud/ocis/issues/400 . This issue is mainly for `the public uses the webUI to access the last public link created by user... ` line in our tests. So, other descriptions about why scenario is skipped will be done in another PR.